### PR TITLE
Resolve #7 Fix Financial Information/Calculations in 5.D.1.C

### LINKED ISSUE #7: Fix Financial Information/Calculations in 5.D.1.C
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:

- The table in 5.D.1.C says 10% is reserved for Accum, however only 5% is. 
- The total operational budget calculation is incorrect.
- There are 55 on-floor members, not 72.

Additionally, it may be worth discussing removing the "suggested operational budget" section.

### PR DESCRIPTION
**Pull request by Sam-Cordry**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
This PR proposes a change to 5.D.1.C, namely by removing all calculations and clarifying where dues from on-floor and off-floor members go.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -505,9 +505,7 @@
 \end{tabular}
 \end{center}
 
-The suggested total operational budget is \$8064.
-This figure is based on yearly estimated on-floor member dues (\$160 x 72 members = \$11520) minus the 10\% reserved for Accumulated (Accum).
-Money allocated for Accum and off-floor member dues is deposited into the CSH Account and saved.
+Dues collected from on-floor members are to be distributed into directorship budgets as shown in the above table. Money allocated for Accumulated and money collected from off-floor members is deposited into the CSH Account and saved.
```
Comment (rhochgraf21) [inline constitution.tex:508]: What do you think about something like this: "Money allocated for Accumulated or collected from off-floor member dues is deposited into the CSH Account and saved"?

File: constitution.tex [hunk 2]
```diff
@@ -505,9 +505,7 @@
 \end{tabular}
 \end{center}
 
-The suggested total operational budget is \$8064.
```
Comment (rhochgraf21) [inline constitution.tex:508]: If we remove these lines from the document, I think we could add some of their information in the LaTeX comments.

The budget info I'd recommend in such a comment would be:
- suggested operational budget of $8360
- ($160 x 55 on-floor members), minus 5% for Accumulated

File: constitution.tex (hunk 1 above)
Comment (rhochgraf21) [inline constitution.tex:508]: General formatting practice is one line per sentence.

Comment (Sam-Cordry) [reply constitution.tex]: I do like that way of phrasing that. It's been changed in the most recent commit.

File: constitution.tex (hunk 2 above)
Comment (Sam-Cordry) [reply constitution.tex:508]: Yes, very good point. I've put a couple lines of comments in there to reflect that info.

File: constitution.tex (hunk 1 above)
Comment (Sam-Cordry) [reply constitution.tex]: Good catch, and it's been fixed now.

**Review by rhochgraf21 — APPROVED** [review]
lgtm

**Pull request merged**